.github/jobs/configure-checks/distclean.bats

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env bats
```

Comment:
It looks like this file shares a lot of duplicate code with `all.bats`. Can common functionality be extracted?